blocks/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * decorates the breadcrumbs
+ * @param {Element} block The breadcrumbs block element
+ */
+const decorateBreadCrumbs = (block) => {
+  [...block.children].forEach((singleItem, index, arr) => {
+    const linkTag = singleItem.querySelector('a');
+    linkTag.classList.add('breadcrumb-link');
+    // Make the last item of the breadcrumb as the active one
+    if (index === arr.length - 1) {
+      linkTag.classList.add('active');
+    }
+    const breadcrumbTitle = linkTag.innerText;
+    linkTag.innerText = '';
+    const breadcrumbItemDiv = document.createElement('div');
+    breadcrumbItemDiv.className = 'breadcrumb-title';
+    breadcrumbItemDiv.innerText = breadcrumbTitle;
+    linkTag.appendChild(breadcrumbItemDiv);
+  });
+};
+
+/**
+ * decorates the breadcrumbs, mainly the body's top breadcrumbs
+ * @param {Element} block The breadcrumbs block element
+ */
+export default async function decorate(block) {
+  decorateBreadCrumbs(block);
```

Comment:
Nit: Maybe all code can move to this single line function :)

Reply:
Yes done, looks cleaner now :)